src/main/java/com/aerospike/mapper/tools/AeroMapper.java

```diff
@@ -77,6 +83,71 @@ public Builder preLoadClass(Class<?> clazz) {
             return this;
         }
 
+        public Builder preLoadClasses(Class<?> ... clazzes) {
+            if (classesToPreload == null) {
+                classesToPreload = new ArrayList<>();
+            }
+            classesToPreload.addAll(Arrays.asList(clazzes));
+            return this;
+        }
+        
+        public String getPackageName(Class<?> clazz) {
+        	Class<?> c;
+        	if (clazz.isArray()) {
+        		c = clazz.getComponentType();
+        	}
+        	else {
+        		c = clazz;
+        	}
+        	String pn;
+        	if (c.isPrimitive()) {
+        		pn = "java.lang";
+        	}
+        	else {
+                String cn = c.getName();
+                int dot = cn.lastIndexOf('.');
+                pn = (dot != -1) ? cn.substring(0, dot).intern() : "";
+        	}        		
+            return pn;
+        }
+
+
+        public Builder preLoadClassesFromPackage(Class<?> classInPackage) {
+        	return preLoadClassesFromPackage(getPackageName(classInPackage));
+        }
+
+        public Builder preLoadClassesFromPackage(String thePackage) {
+        	Set<Class<?>> clazzes = findAllClassesUsingClassLoader(thePackage);
+        	for (Class<?> thisClazz : clazzes) {
+        		// Only add classes with the AerospikeRecord annotation.
+        		if (thisClazz.getAnnotation(AerospikeRecord.class) != null) {
+        			this.preLoadClass(thisClazz);
+        		}
+        	}
+        	return this;
+        }
+        
+        // See https://www.baeldung.com/java-find-all-classes-in-package
+	    private Set<Class<?>> findAllClassesUsingClassLoader(String packageName) {
```

Comment:
Too many spaces/tabs - formatter will fix that ("⌘⌥L (macOS), or Ctrl+Alt+L (Windows/Linux) to reformat a selection of code")